pkg/kubelet/eviction/helpers.go

```diff
@@ -367,6 +367,16 @@ func parseThresholdStatements(statements map[string]string) ([]evictionapi.Thres
 			results = append(results, *result)
 		}
 	}
+	if utilfeature.DefaultFeatureGate.Enabled(features.InheritDefaultEvictionValues){
```

Comment:
Please add a unit test around this code.

Reply:
@kannon92 I have uploaded the code so that it can be reviewed and I am working on the test cases. For now, it is expected that test cases under /pkg/kubelet/eviction/helpers_test.go might fail.